seqr/views/utils/variant_utils.py

```diff
@@ -83,3 +89,108 @@ def saved_variant_genes(variants):
         if gene:
             gene['locusListGuids'] = []
     return genes
+
+
+def _add_locus_lists(projects, genes, add_list_detail=False, user=None, is_analyst=None):
+    locus_lists = LocusList.objects.filter(projects__in=projects)
+
+    if add_list_detail:
+        locus_lists_by_guid = {
+            ll['locusListGuid']: dict(intervals=[], **ll)
+            for ll in get_json_for_locus_lists(locus_lists, user, is_analyst=is_analyst)
+        }
+    else:
+        locus_lists_by_guid = defaultdict(lambda: {'intervals': []})
+    intervals = LocusListInterval.objects.filter(locus_list__in=locus_lists)
+    for interval in _get_json_for_models(intervals, nested_fields=[{'fields': ('locus_list', 'guid')}]):
+        locus_lists_by_guid[interval['locusListGuid']]['intervals'].append(interval)
+
+    for locus_list_gene in LocusListGene.objects.filter(locus_list__in=locus_lists, gene_id__in=genes.keys()).prefetch_related('locus_list', 'palocuslistgene'):
+        gene_json = genes[locus_list_gene.gene_id]
+        locus_list_guid = locus_list_gene.locus_list.guid
+        gene_json['locusListGuids'].append(locus_list_guid)
+        if hasattr(locus_list_gene, 'palocuslistgene'):
+            if not gene_json.get('locusListConfidence'):
+                gene_json['locusListConfidence'] = {}
+            gene_json['locusListConfidence'][locus_list_guid] = locus_list_gene.palocuslistgene.confidence_level
+
+    return locus_lists_by_guid
+
+
+def _get_rna_seq_outliers(gene_ids, families):
```

Comment:
copy-pasted from saved_variant_api.py, no changes other than making it private to this file and swapping out the `sample_filter` argument for the `families` argument, as now all calls to it use the same filtering